### PR TITLE
[homematic] Fix long button press handling for HM-IP devices

### PR DESCRIPTION
**Pull request by maniac103**
HM-IP(W) devices seem to use a different format for long press handling.

(My) HM devices does this:
    PRESS_CONT
    PRESS_LONG
    PRESS_CONT (N times for repetion)
    PRESS_LONG_RELEASE

while (at least) HmIPW-DRI32 does this (as determined [here](https://community.openhab.org/t/homematic-binding-cant-set-value-for-datapoint-warning-for-master-parameter/42145/20)):
    PRESS_LONG
    PRESS_LONG_START
    PRESS_LONG (N times for repetition)
    PRESS_LONG_RELEASE

Add support for the HMIP format while keeping support for the HM format.

### CODE REVIEW

File: bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/virtual/ButtonVirtualDatapointHandler.java
Comment (maniac103) [inline bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/virtual/ButtonVirtualDatapointHandler.java:39]: Probably need a better solution here, as the handler is a singleton. Either a `Map<HmDevice, Boolean>` or some other way to suppress the start button press timeout handling (LONG on HM, LONG_START on HMIP) without suppressing the repetition timeout handling (which is LONG on HMIP).

**Review by lolodomo — APPROVED** [review]
Code LGTM

**Comment by lolodomo** [comment]
I let a chance to binding maintainers to review before merging. 
@FStolte @gerrieg @mdicke2s

**Comment by lolodomo** [comment]
As there is no veto from binding authors, I merge the PR.

**Pull request merged**
